tectonic/xetexini.c

```diff
@@ -3736,7 +3736,7 @@ tt_run_engine(char *dump_name, char *input_file_name)
     /* Make this something invariant so that we can use XDV files to test
      * reproducibility of the engine output. */
 
-    output_comment = "tectonic";
+    output_comment = xstrdup("tectonic");
```

Comment:
No, *currently*, it is not possible. In [engine-interface.c](https://github.com/tectonic-typesetting/tectonic/blob/master/tectonic/engine-interface.c#L38), `output_comment` is allocated. Only here it is a constant. Thus, I use `xstrdup`.

I think it could be fixed but differently, like not using a global variable. However, it requires more significant changes. Another PR improving the code by removing global variables would be more appropriate.